flake8_print.py

```diff
@@ -1,54 +1,38 @@
-import ast
-import tokenize
+"""Extension for flake8 that finds usage of print."""
+import re
 
-from sys import stdin
-
-__version__ = '1.6.1'
+__version__ = '1.7.0'
```

Comment:
I don't follow it to the letter, but I use it as a guideline. I was planning on upping it to 2.